### PR TITLE
PAL Initial framework

### PR DESCRIPTION
**Pull request by cl0w5**
* README
* List of auditors
* How to apply
* Criteria for applications
* PAL internal expenses framework

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# pal-docs
\ No newline at end of file
+# pal-docs
```
Comment (albertov19) [inline README.md]:
```suggestion
# PAL Docs
```

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+This repo contains all relevant documents which define the functioning of the Polkadot Assurance Legion.
```
Comment (albertov19) [inline README.md]:
```suggestion
This repo contains all relevant documents which define the functioning of the [Polkadot Assurance Legion](https://polkadotassurance.com/) (PAL).
```

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+* [List of auditors](/auditors.md)
+* [How to apply for an audit](/applications_howto.md)
+* [Criteria for audit applications](/applications_criteria.md)
+* [Apply for audit funding with PAL](https://docs.google.com/forms/d/e/1FAIpQLSfhF6TNyDVHm7LpcmXl1ydEiRXZ378l_4cNnkEdVeLz8Kt4ag/viewform)
+
+## Other PAL documents
+* [Framework for covering internal PAL expenses](/curators_expenses.md)
+
+## About PAL
+The Polkadot Assurance Legion (PAL) is an initiative of the Polkadot community which aims to increase the auditing capacity of the ecosystem. It has received funds from the Polkadot treasury (a bounty) which will be used to help selected consumers of blockspace undergo audits, and to onboard more auditors to Rust / Polkadot SDK. The ultimate goal is to make Polkadot a more secure and accessible ecosystem to build in.
```
Comment (albertov19) [inline README.md]:
```suggestion
The [Polkadot Assurance Legion](https://polkadotassurance.com/) (PAL) is an initiative of the Polkadot community which aims to increase the auditing capacity of the ecosystem. It has received funds from the Polkadot treasury (through [a bounty](https://polkadot.polkassembly.io/referenda/47)) which will be used to help selected consumers of blockspace undergo audits, and to onboard more auditors to Rust / Polkadot SDK. The ultimate goal is to make Polkadot a more secure and accessible ecosystem to build in.
```

File: applications_criteria.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+## Eligibility Criteria
+
+In order to be eligible for funding under PAL, every application must cover the complete list of eligibility criteria:
+
+1. The applicant must be a parachain which is producing blocks on the Polkadot network;
+2. The application must specify which Substrate pallets or ink! smart contracts would be audited including refs to GitHub and a summary of the total number lines of code;
+3. The code must be open source;
+4. The code must be production-ready and under development-freeze - no new commits are allowed unless agreed with the auditors (fixes of identified bugs);
+5. The Polkadot Audit Fund will cover a maximum of 80% of the auditing costs up to a limit of 18,000 DOT. The applicant parachain must be ready to cover the remaining costs;
+6. The applicant parachain will receive the funding from the bounty mechanism in DOT. The applicant may need to handle the payment to the auditor directly in stablecoins or fiat.
```
Comment (albertov19) [inline applications_criteria.md]:
```suggestion
6. The applicant parachain will receive the funding from the bounty mechanism in DOT. The applicant may need to handle the payment to the auditor directly in whatever form is required by the auditor
```

**Review by albertov19 — APPROVED** [review]
Looks good, left some minor suggestions but looks good as is.

File: applications_criteria.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+## Criteria for Prioritisation
+
+If there are more applications under PAL in a given round than the available funding, the eligible applications will be subjected to the following criteria to determine priority:
+
+1. Count of active wallets;
+2. Count of open bi-directional HRMP/XCM channels;
```
Comment (xlc) [inline applications_criteria.md]:
```suggestion
2. Count of open and active bi-directional HRMP/XCM channels;
```

**Review by xlc — APPROVED** [review]

**Review by PierreOssun — APPROVED** [review]

**Pull request merged**